core/core.go

```diff
@@ -93,6 +108,10 @@ type Block struct {
 	Sha256Hash string `json:"sha256hash"`
 	// Location of the provider where the block is stored
 	Location BlockLocation `json:"location"`
+	// Audits to be used to check the block's integrity
+	Audits []BlockAudit `json:"audits"`
+	// Whether or not the last audit passed.
+	AuditPassed bool `json:"auditPassed"`
```

Comment:
I'm not a big fan of keeping this with the block itself.

Reply:
I also thought it might make it easier for the renter to use that data to restore blocks.